test/lib.ResultSet.js

```diff
@@ -338,6 +338,35 @@ describe('Lib', function () {
       });
 
     it(
+      'should fetch all rows with a LOB and rowsAsArray set',
```

Comment:
I think the formatting is such so that the max line character length in approximately 80.
If you look at the ones that are properly formatted you will notice that they are below 80; however the longer ones that exceed 80 are split into multiple lines.